galley/pkg/fs/fssource_test.go

```diff
@@ -326,16 +327,17 @@ func TestFsSource_DeletePartResorceInFile(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Unexpected error: %v", err)
 	}
-
-	log := logChannelOutput(ch, 4)
+	//In some OS, it will trigger file deletion event
```

Comment:
Can you add more details? (i.e. which O.S., what is the expected event sequence v.s. what is received etc.)

Thanks

Reply:
done